packages/bootstrap/scss/_variables.scss

```diff
@@ -316,6 +316,11 @@ $popup-border: $dropdown-border-color !default;
 $popup-shadow: 0 2px 4px 0 rgba(0, 0, 0, .03), 0 4px 5px 0 rgba(0, 0, 0, .04) !default;
 
 
+// Grouping
+$grouping-header-text: $bg-color !default;
+$grouping-header-bg: $text-color !default;
+
```

Comment:
For consistency with other themes, I would suggest adding the `$grouping-border: $border-color !default;` variable, and using it on lines 22 and 25 of `bootstrap/scss/popup/_theme.scss`